test/powershell/engine/Help/UpdatableHelpSystem.Tests.ps1

```diff
@@ -308,7 +380,19 @@ Describe "Validate Update-Help from the Web for all PowerShell Core modules." -T
         $ProgressPreference = $SavedProgressPreference
     }
 
-    RunUpdateHelpTests -tag "Feature"
+    RunUpdateHelpTests -tag "Feature" -Scope 'AllUsers'
+}
+
+Describe "Validate Update-Help from the Web for all PowerShell Core modules for user scope." -Tags @('Feature', 'RequireAdminOnWindows', 'RequireSudoOnUnix') {
+    BeforeAll {
+        $SavedProgressPreference = $ProgressPreference
+        $ProgressPreference = "SilentlyContinue"
+    }
+    AfterAll {
+        $ProgressPreference = $SavedProgressPreference
+    }
+
+    RunUpdateHelpTests -tag "Feature" -Scope 'CurrentUser'
```

Comment:
@adityapatwardhan @SteveL-MSFT  I wonder why we use `-Scope` if the function have only `[switch]$userscope`?

Reply:
From a quick look seems to be a bug in the test.